lwt-unix/conduit_lwt_unix.ml

```diff
@@ -181,8 +179,7 @@ module Sockaddr_server = struct
      with
      (* This is expected for Unix domain sockets *)
      | Unix.Unix_error(Unix.EOPNOTSUPP, _, _) -> ());
-    let ic = Lwt_io.of_fd ~mode:Lwt_io.input client in
-    let oc = Lwt_io.of_fd ~mode:Lwt_io.output client in
+    Lwt_io.open_connection ~fd:client peeraddr >>= fun (ic, oc) ->
```

Comment:
There is no `Lwt_unix.connect` here but `Lwt_io.open_connection` has a call to connect, as mentioned above.  I'm not sure what issues this would cause.